hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/dao/GatewayServiceEntityDao.java

```diff
@@ -48,9 +52,10 @@
   public Single<EntityResultSet> getEntities(EntityRequest request) {
     return this.requestBuilder
         .buildRequest(request)
-        .flatMap(
-            serverRequest -> this.makeRequest(request.resultSetRequest().context(), serverRequest))
-        .flatMap(serverResponse -> this.entityConverter.convert(request, serverResponse));
+        .flatMap(serverRequest ->  this.makeRequest(request.resultSetRequest().context(), serverRequest)
+        .flatMap(serverResponse -> baselineProvider.buildRequest(serverRequest, serverResponse, request)
```

Comment:
this indentation is off, hiding the nesting. Please update, and break into multiple functions if the nesting seems confusing.